docs/api/saved-objects/import.asciidoc

```diff
@@ -109,13 +190,69 @@ The API returns the following:
       "title": "my-pattern-*",
       "error": {
         "type": "conflict"
-      },
+      }
+    },
+    {
+      "id": "my-visualization",
+      "type": "my-vis",
+      "title": "Look at my visualization",
+      "error": {
+        "type": "conflict",
+        "destinationId": "another-vis"
+      }
     },
+    {
+      "id": "my-canvas",
+      "type": "canvas-workpad",
+      "title": "Look at my canvas",
+      "error": {
+        "type": "ambiguous_conflict",
+        "destinations": [
+          {
+            "id": "another-canvas",
+            "title": "Look at another canvas",
+            "updatedAt": "2020-07-08T16:36:32.377Z"
+          },
+          {
+            "id": "yet-another-canvas",
+            "title": "Look at yet another canvas",
+            "updatedAt": "2020-07-05T12:29:54.849Z"
+          }
+        ]
+      }
+    }
   ],
+  "successResults": [
+    {
+      "id": "my-dashboard",
+      "type": "dashboard"
+    }
+  ]
 }
 --------------------------------------------------
 
-Import a visualization and dashboard with an index pattern for the visualization reference that doesn't exist:
+This result indicates that the import was not successful because the index pattern, visualization, and dashboard each resulted in a conflict error:
+
+* An index pattern with the same ID already exists, so this resulted in a conflict error. This can be resolved by overwriting the existing
+object, or skipping this object entirely.
+
+* A visualization with a different ID but the same "origin" already exists, so this resulted in a conflict error as well. The
+`destinationId` field describes to the other visualization which caused this conflict. This behavior was added to ensure that new objects
+which can be shared between spaces behave in a similar way as legacy non-shareable objects. When a shareable object is exported and then
+imported into a new space, it retains its origin so that it conflicts will be encountered as expected. This can be resolved by overwriting
+the specified destination object, or skipping this object entirely.
+
+* Two canvases with different IDs but the same "origin" already exist, so this resulted in an ambiguous conflict error. The `destinations`
+array describes to the other canvases which caused this conflict. When a shareable object is exported and then imported into a new space,
+and is _then_ shared to another space where an object of the same origin exists, this situation may occur. This can be resolved by picking
+one of the destination objects to overwrite, or skipping this object entirely.
```

Comment:
I mention Spaces here -- do we need to specify that this is only applicable to installations using X-Pack? If so, how should we specify that?

Reply:
What do you think about linking to [the spaces docs](https://github.com/elastic/kibana/blob/896b9cddbc93b4f35d3c41624e5d3dd4714703b4/docs/spaces/index.asciidoc#L1) inline? That would at least give us a reference to docs which reference x-pack, without distracting from your example here. Something like:

`.... _then_ shared to another <<xpack-spaces,space>> where an object of the same origin exists, ...`

Reply:
Good idea! Done in dd25fbc, added in the first section and also at the first mention of spaces in that example.